### PR TITLE
Fidels/normalization fix

### PR DESCRIPTION
**Pull request by fidel-schaposnik**
We have to make sure that the loss computation does not independently normalize logits for global and local arguments!

### CODE REVIEW

**Comment by LasseBlaauwbroek** [comment]
Do you think this improves the global argument predictions?
